controllers/dnsupdate.go

```diff
@@ -166,3 +161,19 @@ func (r *GslbReconciler) updateRuntimeStatus(gslb *k8gbv1beta1.Gslb, isPrimary b
 		m.UpdateFailoverStatus(gslb, isPrimary, isHealthy, finalTargets)
 	}
 }
+
+// getLabels map of where key identifies region and weight, value identifies IP.
+func (r *GslbReconciler) getLabels(gslb *k8gbv1beta1.Gslb, targets assistant.Targets) (labels map[string]string) {
```

Comment:
Shouldn't that be named genLabels instead? getLables quite confusing.

Reply:
Hi @k0da, thx for good comment. I will try to explain: Yes it generates labels and therefore it should be called `genLabels`.  `getLabels` is **getter function** that **always returns** a label map. 

The main reason why I would choose `get` over `gen`. now the main reason why I would choose `get` over `gen`. Overall practice is getters don't change state. A getter returns or generates a value, but it doesn't change the state (for example, if the `getLabels` function somewhere inside would somehow change the endpoint, or gslb etc....). A **getter can't change anything**, just read anything and return the value or modified or generated value. 

This reduces the cognitive load a lot, because if something is a getter, the developer doesn't have to worry about the details of how the get works. He doesn't have to study the body of the function in order to have it change some instances.